libnetwork/osl/namespace_linux.go

```diff
@@ -338,9 +348,7 @@ func createNetworkNamespace(path string, osCreate bool) error {
 
 func unmountNamespaceFile(path string) {
 	if _, err := os.Stat(path); err == nil {
-		if err := syscall.Unmount(path, syscall.MNT_DETACH); err != nil && !errors.Is(err, unix.EINVAL) {
-			logrus.WithError(err).Error("Error unmounting namespace file")
-		}
+		syscall.Unmount(path, syscall.MNT_DETACH)
```

Comment:
Unrelated change, or was there a reason for ignoring the error here?

Reply:
Unrelated. Likely to be a bad rebase, since this was originally written against the "old" libnetwork repo before I realized it was archived and moved this over